### ISSUE TITLE
Add new WS to obtain individual relatives

### ISSUE THREAD

**Issue by pfurio** [opencb/opencga#1552]
It would be really interesting having a new web service that will automatically gather and return all the relatives of an individual. The web service will look like:
`/individuals/{individual}/relatives`
Users will be able to get all the relatives of degree 1 (parents and children) and degree 2 (up to grandparents, grandchildren and brothers/sisters).
Additionally, OpenCGA will add the relation with the main individual in the _attributes_ field.

**Comment by imedina** [comment]
This is a nice feature, just one simple question. Users will only get the relatives they can _VIEW_ right? Assuming the answer is yes, could you add an _event_ type WARNING saying there are some 1st or 2nd degree relatives not returned because the lack of permissions?

**Comment by pfurio** [comment]
Yes, users will only be able to see the relatives for which they have VIEW permission.
I definitely agree it would be great adding warning events for those relatives the user does not have VIEW permission for. However, adding that information is not straightforward so I would postpone that improvement for next 2.1 version.
